src/test/enhetstester/kotlin/no/nav/familie/ba/sak/common/DbContainerInitializer.kt

```diff
@@ -23,7 +23,7 @@ class DbContainerInitializer : ApplicationContextInitializer<ConfigurableApplica
     companion object {
         // Lazy because we only want it to be initialized when accessed
         private val postgres: KPostgreSQLContainer by lazy {
-            KPostgreSQLContainer("postgres:12.10")
+            KPostgreSQLContainer("postgres:15.3")
```

Comment:
Vi kjører 12 i prod. Bør kanskje oppgradere der først?

Reply:
Ruller tilbake denne, så kan vi endre denne separat
```suggestion
            KPostgreSQLContainer("postgres:12.10")
```